File: launchdarkly/python/src/GetExperimentExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.ExperimentsApi(api_client).get_experiment(
            project_key=None,
            environment_key=None,
            experiment_key=None,
            expand=None,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling ExperimentsApi#get_experiment: %s\n" % e)

File: launchdarkly/typescript-axios/src/GetExperimentExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.ExperimentsApi(configuration).getExperiment(
  "projectKey_string", // projectKey
  "environmentKey_string", // environmentKey
  "experimentKey_string", // experimentKey
  undefined, // expand
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling ExperimentsApi#getExperiment:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
